Vagrantfile

```diff
@@ -0,0 +1,68 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+hostname      = 'st2-packages'
+st2user       = ENV['ST2USER'] ? ENV['ST2USER']: 'st2admin'
+st2passwd     = ENV['ST2PASSWORD'] ? ENV['ST2PASSWORD'] : 'Ch@ngeMe'
+box           = ENV['BOX'] ? ENV['BOX'] : 'centos/7'
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+VIRTUAL_MACHINES = {
+  :ubuntu14 => {
+    :hostname => 'ubuntu14',
+    :box => 'ubuntu/trusty64',
+    :target => 'trusty',
+  },
+  :ubuntu16 => {
+    :hostname => 'ubuntu16',
+    :box => 'ubuntu/xenial64',
+    :target => 'xenial',
+  },
+  :centos6 => {
+    :hostname => 'centos6',
+    :box => 'centos/6',
+    :target => 'el6',
+  },
+  :centos7 => {
```

Comment:
Yes, agreed. Thanks for the comments. I have also removed the global `hostname` and `box` variables as they aren't used anymore.